### ISSUE TITLE
CheatTable converter? 

### ISSUE THREAD

**Issue by LegitSoulja** [manovisible/lunarengine#3]
I noticed some `Cheat` related terms was changed to `Lunar`, though is this the only necessary change? 
If so, I plan to write a quick program (maybe web) to convert tables to Lunar.

Thanks in advance!.

Edit: Converter => https://legitsoulja.github.io/CT2LT/
Source: https://legitsoulja.github.io/CT2LT/ct2lt.js

**Comment by manovisible** [comment]
Yes, that’s exactly what I did. It will bypass every game anti-cheat that scans for string value (aka the word “cheat”). You can even change the word “Engine” for any word that has the same amount of letters :)

**Comment by LegitSoulja** [comment]
> Yes, that’s exactly what I did. It will bypass every game anti-cheat that scans for string value (aka the word “cheat”). You can even change the word “Engine” for any word that has the same amount of letters :)

That's not exactly the answer I was looking for, though I know this makes the engine "undetected".
I think I've managed to make a successful converter, after testing numerous other tables.

**Comment by LegitSoulja** [comment]
> Yes, that’s exactly what I did. It will bypass every game anti-cheat that scans for string value (aka the word “cheat”). You can even change the word “Engine” for any word that has the same amount of letters :)

That's not exactly what I asked, and obviously this makes the engine "undetected".
I've managed to make a successful converter, after testing numerous other tables. I found all I was looking for, even for the functions.

BTW "Engine" doesn't need to be changed, and doesn't have to be replaced with the same number of characters.. Irrelevant to my original post, but thanks.
